packages/enterprise/src/license.ts

```diff
@@ -664,6 +664,10 @@ async function updateLicenseFromServer(
       logger.error(`Error creating mongo cache: ${e}`);
       throw e;
     });
+    verifyLicenseInterface(license);
+    keyToLicenseData[licenseKey] = license;
+    keyToCacheDate[licenseKey] = new Date();
+    throw e;
```

Comment:
Is this change intentional?  Seems unrelated to setting cloud licenses via admin

Reply:
Well it is related in the sense that it was a bug I found when the test plan didn't work as expected.  It should have been its own PR though.  But now this code has a merge conflict with https://github.com/growthbook/growthbook/pull/2413.  So I think I would like to land that one first.  Then I'll make a new PR that fixes this bug, then I'll update this PR removing this code.